income.json

```
@@ -39,9 +41,12 @@
       "chance": 60,
       "success": [
         "А вы прям прирожденный вор! Украли целых %amount%"
+        "Вы беспалева ограбили Пятёрочку и заработали %amount%"
       ],
       "fail": [
         "Не повезло, не повезло... Вас поймали! Вы потеряли %amount%"
+        "Пока вы грабили банк, приехали менты и отштрафовали вас на %amount%"
```

Не менты, а полицейские!
```suggestion
        "Пока вы грабили банк, приехали полицейские и оштрафовали вас на %amount%",
```